.travis.yml

```
@@ -36,6 +37,13 @@ jobs:
         - BUILD=unit
         - BUILD_TYPE=Release
         - CXXFLAGS="-std=c++2a"
+    - name: "Unit gcc10 (c++17)"
+      env:
+        - CXX=g++-10
+        - CC=gcc-10
+        - BUILD=unit
+        - BUILD_TYPE=Debug
```

```suggestion
        - BUILD_TYPE=Release
```

Cached debug builds still need 15 minutes in contrast to the 2 minutes release builds need (time for running tests).
We also have the Debug builds in nightly.